larpix/larpix.py

```diff
@@ -1566,6 +1567,120 @@ def save_output(self, filename, message):
             json.dump(data, outfile, indent=4,
                     separators=(',',':'), sort_keys=True)
 
+class TimestampPacket(object):
```

Comment:
Currently the timestamps are generated by the daq boards individually - there should be an additional field to track this information.

Reply:
I'm not sure exactly what you mean here. Are you suggesting a field such as ``TimestampPacket.source`` that labels timestamps by the DAQ board that generated them, for when we use multiple DAQ boards? In that situation, there should probably be a "single source of truth" which eventually will be the GPS clock, but in the short term could just be one particular DAQ board that sends timestamps, while the others do not. So then there would be no ambiguity about where the timestamps are coming from.

Thoughts? Am I interpreting your suggestion correctly?

Reply:
I guess from my discussions with Dan, I was thinking that the GPS timestamp was received by the DAQ board firmware and then injected into the datastream at that point. If the GPS timestamp is injected into the network at a different point, I think there might be timing issues associated with network delays.
We should probably ask Dan for more clarity on this.

Reply:
Discussion with @dadwyer today about timestamps - each DAQ board will issue timestamps so I retract my suggestion. I will implement a field or fields in ``TimestampPacket`` to identify the source.